File: genetic-car-starters/genetic-car-starter-java/src/main/java/fr/genetic/client/java/api/CarScoreView.java
package fr.genetic.client.java.api;

public class CarScoreView {
    public CarView car;
    public float score;

    @Override
    public String toString() {
        return "CarScoreView{" +
                "car=" + car +
                ", score=" + score +
                '}';
    }
}

File: genetic-car-starters/genetic-car-starter-csharp/GeneticAlgorithm/Api/CarScoreView.cs
namespace GeneticAlgorithm.Api
{
    public class CarScoreView
    {
        public CarView Car { get; set; }
        public float Score { get; set; }

        public override string ToString()
        {
            return "CarScoreView{" +
                    "car=" + Car +
                    ", score=" + Score +
                    '}';
        }

    }
}
